### ISSUE TITLE
build plasma-wayland-protocols failed 

### ISSUE THREAD

**Issue by dassio** [cgilles/digikam-install-deps#2]
when running `sudo ./03-linux-installkf6.sh`, got the following error: 

```
[ 50%] Performing download step (git clone) for 'ext_plasma-wayland-protocols'
Cloning into 'ext_plasma-wayland-protocols'...
fatal: invalid reference: v6.2.0
CMake Error at /mnt/data/build.qt6/ext_kf6/ext_plasma-wayland-protocols-prefix/tmp/ext_plasma-wayland-protocols-gitclone.cmake:49 (message):
  Failed to checkout tag: 'v6.2.0'
```

**Comment by dassio** [comment]
after changing to v1.13.0 for `plasma-wayland-protocols`,  

```
danny@digikam-dev-24-latest:~/programming/digikam-install-deps$ git diff
diff --git a/3rdparty/ext_kf6/CMakeLists.txt b/3rdparty/ext_kf6/CMakeLists.txt
index d44e158..6b4049c 100755
--- a/3rdparty/ext_kf6/CMakeLists.txt
+++ b/3rdparty/ext_kf6/CMakeLists.txt
@@ -24,6 +24,7 @@ function(InstallKF6Component
             set(giturl "https://invent.kde.org/libraries/${name}.git")
         elseif(${name} STREQUAL "plasma-wayland-protocols")
             set(giturl "https://invent.kde.org/libraries/${name}.git")
+            set(version "v1.13.0")
         elseif(${name} STREQUAL "breeze")
             set(giturl "https://invent.kde.org/plasma/${name}.git")
         else()
```


and continue installation , similar error : 


```
[100%] Completed 'ext_ktextwidgets'
[100%] Built target ext_ktextwidgets
[  0%] Creating directories for 'ext_qca'
[ 50%] Performing download step (git clone) for 'ext_qca'
Cloning into 'ext_qca'...
fatal: invalid reference: v6.2.0
CMake Error at /mnt/data/build.qt6/ext_kf6/ext_qca-prefix/tmp/ext_qca-gitclone.cmake:49 (message):
  Failed to checkout tag: 'v6.2.0'
```

**Comment by cgilles** [comment]
Fixed with commit #5f1b255f3e5f137a33609777f3e158e6f2ee97aa
